bcgov/mds
# Mds 3922 update minespace antdv4

# Main
- Update Antd 3 to version 4
- Updated all necessary dependencies
- Updated all necessary styling

# How to test
- Pull down branch and run "npm install" on the minespace-web
- Confirm everything looks as it should, all forms function correctly.

# Notes
<img width="1790" alt="Screen Shot 2021-09-29 at 5 01 19 PM" src="https://user-images.githubusercontent.com/25125247/135364508-589ac472-e9d7-4f1c-8da9-f6efcbe49a71.png">
<img width="1791" alt="Screen Shot 2021-09-29 at 5 02 35 PM" src="https://user-images.githubusercontent.com/25125247/135364608-867e0ebe-4666-44ae-a9a3-cac9dbe25685.png">
<img width="1789" alt="Screen Shot 2021-09-29 at 5 02 50 PM" src="https://user-images.githubusercontent.com/25125247/135364621-832b072c-bcce-4b27-9389-cfff090e19b4.png">


<img width="1791" alt="Screen Shot 2021-09-29 at 5 03 32 PM" src="https://user-images.githubusercontent.com/25125247/135364656-7d804e61-1488-4513-ad67-d3336d01c47c.png">


<img width="1792" alt="Screen Shot 2021-09-29 at 5 04 26 PM" src="https://user-images.githubusercontent.com/25125247/135364719-a250e3d2-a72b-4845-9011-842228afe451.png">

<img width="1709" alt="Screen Shot 2021-09-29 at 5 04 55 PM" src="https://user-images.githubusercontent.com/25125247/135364763-3d168e00-893f-4a70-b51d-e5b3dd04d285.png">
